src/ir/block_builder.h

```diff
@@ -0,0 +1,84 @@
+//-----------------------------------------------------------------------------
+// Copyright 2021 Google LLC
```

Comment:
Copyright year.

Reply:
argh, not again. I solemnly promise that I really created this file in 2021. (I actually did! :) )

Jokes aside, I will change the year.